### ISSUE TITLE
[Feat Req] Follow ReplaceInFiles with auto FindInFiles

### ISSUE THREAD

**Issue by alankilborn** [notepad-plus-plus/notepad-plus-plus#13378]
In Visual Studio's text editor, if you do a Replace-in-Files, it automatically does a Find-in-Files on the replacement text, once the replacing is complete.  It might be nice to have this feature as an optional post- _Replace in Files_ action in Notepad++.

Request made when considering N++ behavior in v8.5.1.
